src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -51,6 +51,26 @@ public static ReadOnlyAddressBook getEmptyAddressBook() {
         return tasks.getInternalList();
     }
 
+    /**
```

Comment:
you can find task by index only. theres no implementation for getting the readable task object via name. unless ive overseen one. in that case please explain, dont just state without explaining.